lib/public/components/Log/log.js

```diff
@@ -22,13 +22,13 @@ import { logReplyButton } from './logReplyButton.js';
 import { logText } from './logText.js';
 import { primaryButton } from '../common/primaryButton.js';
 import { logDetailsButton } from './logDetailsButton.js';
+import { attachmentPreviewComponent } from '../../views/Logs/FilePreview/attachmentPreviewComponent.js';
 
 /**
  * Provides formatting for each log field to display that field on the webpage
- * @param {string} authenticationToken token to authenticate api requests. Used for attachments
  * @return {Object} an object determining how each log field wil lbe displayed on the page
  */
-const logFields = (authenticationToken) => ({
+const logFields = () => ({
```

Comment:
authenticationToken is removed so is provisioning can be as well
see e.g. lib/public/components/Log/logDetails.js